### PR TITLE
fixed the bug but also had to update the webpack to run with new crea…

### PR DESCRIPTION
**Pull request by sarnikbixal**
…te react app v2 as it was broken when npm modules were updated

Thanks for submitting a pull request! Below are a few things you can do to help us more quickly review your changes.

## Checklist

I have…

- [ ] run the application locally (`make up`) and verified that my changes behave as expected.
- [ ] run static behat test suite (`circleci build --job behat`) against my changes.
- [ ] run the code sniffer (`circleci build --job code-sniffer`) against my changes.
- [ ] run the code coverage tool (`circleci build --job code-coverage`) 
      against my changes
- [ ] summarized below my changes and noted which issues (if any) this pull request fixes or addresses.
- [ ] thoroughly outlined below the steps necessary to test my changes.
- [ ] attached screenshots illustrating relevant behavior before and after my changes.
- [ ] read, understand, and agree to the terms described in [CONTRIBUTING.md](https://github.com/Bixal/move.mil/blob/master/CONTRIBUTING.md).
- [ ] added my name, email address, and copyright date to [CONTRIBUTORS.md](https://github.com/Bixal/move.mil/blob/master/CONTRIBUTORS.md).

## Summary of Changes

This pull request…

- _Summarize change number one._
- _Summarize change number two._
- _etc. etc. etc._

## Testing

To verify the changes proposed in this pull request…

1. _Describe testing step one._
1. _Describe testing step two._
1. _etc. etc. etc._

## Screenshots

_Attach relevant before and after screenshots here._

### CODE REVIEW

File: web/modules/custom/react_tools/tools/apps/react-ppm-tool/src/App.js [hunk 1]
```diff
@@ -154,6 +157,7 @@ class App extends Component {
     axios.post(url, options)
       .then(res => {
         let results = res.data;
+        console.log(results)
```
Comment (g3r4) [inline web/modules/custom/react_tools/tools/apps/react-ppm-tool/src/App.js]: @sarnikbixal Do we want to leave this here?

Comment (sarnikbixal) [reply web/modules/custom/react_tools/tools/apps/react-ppm-tool/src/App.js]: No, good catch.

**Comment by ivonne-gov** [comment]
@sarnikbixal  hey Justin the behat check is failing because `npm test` is failing. Is there a reason why you remove the testing?

> move-mil-react-apps@0.1.0 test /var/www/html/web/modules/custom/react_tools/tools
> node scripts/test.js

No tests found, exiting with code 1
Run with `--passWithNoTests` to exit with code 0
In /var/www/html/web/modules/custom/react_tools/tools
  67 files checked.
  testMatch: /var/www/html/web/modules/custom/react_tools/tools/src/**/__tests__/**/*.{js,jsx,ts,tsx}, /var/www/html/web/modules/custom/react_tools/tools/src/**/*.{spec,test}.{js,jsx,ts,tsx} - 0 matches
  testPathIgnorePatterns: /node_modules/ - 67 matches
  testRegex:  - 0 matches
Pattern:  - 0 matches
npm ERR! Test failed.  See above for more details.

Exited with code exit status 1

**Comment by sarnikbixal** [comment]
> @sarnikbixal hey Justin the behat check is failing because `npm test` is failing. Is there a reason why you remove the testing?
> 
> > move-mil-react-apps@0.1.0 test /var/www/html/web/modules/custom/react_tools/tools
> > node scripts/test.js
> 
> No tests found, exiting with code 1
> Run with `--passWithNoTests` to exit with code 0
> In /var/www/html/web/modules/custom/react_tools/tools
> 67 files checked.
> testMatch: /var/www/html/web/modules/custom/react_tools/tools/src/**/**tests**/**/_.{js,jsx,ts,tsx}, /var/www/html/web/modules/custom/react_tools/tools/src/**/_.{spec,test}.{js,jsx,ts,tsx} - 0 matches
> testPathIgnorePatterns: /node_modules/ - 67 matches
> testRegex: - 0 matches
> Pattern: - 0 matches
> npm ERR! Test failed. See above for more details.
> 
> Exited with code exit status 1

@ivonne-gov I did not intentionally remove any tests. I didn't know we had any custom tests written for this. I think there were only the jest tests that come with create-react-app. I moved over to create-react-app v2. So whatever tests came with that were brought in. I am not sure why it would fail when finding no tests folders. Perhaps we need to include those folders in apps directory. That or we can remove npm test.

**Pull request merged**